azure-pipelines.yml

```diff
@@ -0,0 +1,96 @@
+trigger:
+  batch: true
+  branches:
+    include:
+      - "*"
+
+pr: none
+
+variables:
+  buildConfiguration: 'release'
+  buildPlatform: 'anycpu'
+
+resources:
+  repositories:
+  - repository: das-platform-building-blocks
+    type: github
+    name: SkillsFundingAgency/das-platform-building-blocks
+    ref: refs/heads/master
+    endpoint: "GitHub (SFA)"
+
+jobs:
+- job: 'CodeBuild'
+  pool:
+    name: 'DAS - Continuous Integration'
+  workspace:
+    clean: all
+  steps:
+  - task: gittools.gitversion-preview.gitversion-task.GitVersion@5
```

Comment:
There's a new UseGitVersion task that we can use as GitVersion is being deprecated https://github.com/SkillsFundingAgency/das-alpha-spikes/blob/master/FATv2/APIM/SFA.DAS.FATv2.Alpha/azure-pipelines.yml#L25

Reply:
Refactored